doc/src/actioncode.md

```diff
@@ -0,0 +1,19 @@
+# Action code
+
+Action code is normal Rust code with the addition of the following special variables:
+
+ * `$1` ... `$n` refer to the respective symbol in the production, numbered
+   from 1 (i.e. `$1` refers to the first symbol in the production). If the
+   symbol references a rule `R` then an instance of `R`'s type will be stored
+   in the `$x` variable; if the symbol references a lexeme then an
```

Comment:
Maybe use `$i` instead of `$x` as that may make it clearer we are talking about a variable from `$1...$n`.

Reply:
Agreed.